### PR TITLE
Add validation

### PR DESCRIPTION
**Pull request by pavlokolodka**

### CODE REVIEW

File: src/utils/validator/schemes.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    lastname: {
+      type: "string",
+      maxLength: 256,
+      minLength: 3
+    },
+    password: {
+      type: "string",
+      format: "password",
+      minLength: 8
```
Comment (lerik1408) [inline src/utils/validator/schemes.ts:27]: 👀

Comment (lerik1408) [inline src/utils/validator/schemes.ts:27]: 👀

**Review by lerik1408 — COMMENTED** [review]
GJ!

**Comment by pavlokolodka** [comment]
Separated the schema logic
Resolved a conflict after merge 
Added validation of user update endpoint

File: src/utils/validator/user/password.schema.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { JSONSchemaType } from "ajv";
+
+export const password: JSONSchemaType<string> = {
+  type: "string",
+  format: "password",
+  minLength: 8
+}
```
Comment (lerik1408) [inline src/utils/validator/user/password.schema.ts:7]: Let's move this file to src/utils/validator/shared

File: src/utils/validator/user/loginUser.schema.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export const LoginUserSchema: JSONSchemaType<LoginUserDto> = {
+  type: "object",
+  properties: { 
+    email: email,
+    password: password
+  },
+  required: ["email", "password"],
+  additionalProperties: false
+}
```
Comment (lerik1408) [inline src/utils/validator/user/loginUser.schema.ts:14]: Let's add index file for importing  src/utils/validator/user/index.ts

**Review by lerik1408 — COMMENTED** [review]
GJ

**Review by lerik1408 — COMMENTED** [review]
Fix conflict

**Pull request closed without merging**